subsys/bluetooth/host/Kconfig

```diff
@@ -129,6 +129,35 @@ config BT_DRIVER_RX_HIGH_PRIO
 	int
 	default 6
 
+config BT_CONN_TX_WQ
```

Comment:
The PR mostly looks ok now, so I'll do a nitpick: I would like to rename this to `BT_CONN_TX_NOTIFY_WQ` to avoid confusing it with `tx_work`.

Reply:
Changed